src/WakaTime.Shared.ExtensionUtils/Properties/AssemblyInfo.cs

```diff
@@ -32,5 +32,5 @@
 // You can specify all the values or you can default the Build and Revision Numbers
 // by using the '*' as shown below:
 // [assembly: AssemblyVersion("1.0.*")]
-[assembly: AssemblyVersion("2.1.0")]
-[assembly: AssemblyFileVersion("2.1.0")]
+[assembly: AssemblyVersion("2.1.0.1")]
```

Comment:
I've set the assembly version revision number as a mere suggestion, however, in retrospect, I'm not sure how NuGet managers determine when to auto update a library. Perhaps it will not automatically update if the version number is different, in which case changing the version numer, even if just the revision number, could have been a mistake.

Reply:
We don't bump versions in PRs, let's leave this unchanged.